### PR TITLE
Doc changes to the installation and setup page

### PR DESCRIPTION
**Pull request by adiso75**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -2,24 +2,34 @@
...
+   their own.
+  
+- [Iguazio's Managed  Service](https://www.iguazio.com) - A commerical offering by Iguazio. This is the fastest way to explore the full set of MLRun functionalities.
+  Note that Iguazio provides a 14 days free trial.
+
+
+## Set up your client
+
+- Users can work with their favorite IDE (e.g. Pycharm, VScode, Jupyter , Colab etc..). in this section you'll find how to configure your client against the deployed
+  MLRUN server [How to configure your client](https://docs.mlrun.org/en/latest/install/remote.html)
```
Comment (jillnogold) [inline docs/install.md:26]: You can work with your favorite IDE (e.g. Pycharm, VScode, Jupyter , Colab etc..). Read how to configure your client against the deployed
  MLRun server in [....

File: docs/install.md [hunk 2]
```diff
@@ -2,24 +2,34 @@
...
+- [Local deployment:](https://docs.mlrun.org/en/latest/install/local-docker.html) Deploy a docker on your laptop or on a single server.
+   This option is good for testing the water or when working in a small scale enviroenment. It's limited in terms of compute resources and scale but simpler for
+   deployment.
+
+- [Kubernetes cluster:](https://docs.mlrun.org/en/latest/install/kubernetes.html) Deploy MLrun server on kuberentes.
+   This option deployed MLRun on kubernetes cluster which support elastic scaling yet it is more complex to install as it requires users to install kuberenetes on
+   their own.
+  
+- [Iguazio's Managed  Service](https://www.iguazio.com) - A commerical offering by Iguazio. This is the fastest way to explore the full set of MLRun functionalities.
+  Note that Iguazio provides a 14 days free trial.
```
Comment (jillnogold) [inline docs/install.md:20]: Note that Iguazio provides a 14 day free trial.

**Review by jillnogold — CHANGES_REQUESTED** [review]
In general, and as much as possible, write in first person and present tense.

File: docs/install.md [hunk 3]
```diff
@@ -2,24 +2,34 @@
...
+MLRun comprises of two parts: MLRun Server and MLRUN client.
 
-- Install the MLRun service [locally using Docker](https://docs.mlrun.org/en/latest/install/local-docker.html) or [over Kubernetes Cluster](https://docs.mlrun.org/en/latest/install/kubernetes.html). Alternatively, you can use [Iguazio's managed MLRun service](https://www.iguazio.com/docs/latest-release/).
-- [Set up your client environment](https://docs.mlrun.org/en/latest/install/remote.html) to work with the local or remote MLRun service.
+## Deployment options
+
+
+There are several deployment options:
+- [Local deployment:](https://docs.mlrun.org/en/latest/install/local-docker.html) Deploy a docker on your laptop or on a single server.
+   This option is good for testing the water or when working in a small scale enviroenment. It's limited in terms of compute resources and scale but simpler for
```
[the remark this answers is not in the document]
Comment (adiso75) [reply docs/install.md:12]: for deployment

**Pull request merged**